composer.json

```diff
@@ -24,14 +24,16 @@
         "symfony/event-dispatcher": "^6.0",
         "symfony/http-foundation": "^6.0",
         "symfony/http-kernel": "^6.0",
-        "symfony/options-resolver": "^6.0"
+        "symfony/options-resolver": "^6.0",
+        "webonyx/graphql-php": "^14.11"
```

Comment:
Are you sure it's a correct restriction ? Maybe ^14 would be enough... maybe we should support 0.13 and older versions ?